Human/nbconverted/Human_experiment_lvl_sim.py

```diff
@@ -44,7 +44,7 @@
 warnings.filterwarnings(action='ignore')
 
 sys.path.append("../")
```

Comment:
If you're importing these functions from a different library, you probably don't need to tinker with your pythonpath